app/controllers/users/confirmations_controller.rb

```diff
@@ -2,8 +2,12 @@ module Users
   class ConfirmationsController < Devise::ConfirmationsController
     include ValidEmailParameter
 
+    def new
+      @user = User.new
```

Comment:
I don't have much experience with Devise, just to check my understanding: what this is doing is overriding a `#new` method in Devise that probably assigns to `@confirmable` but now that we're using `@user` we need to do this manually?